src/com/epam/course/hometask3/Main.java

```diff
@@ -0,0 +1,73 @@
+package com.epam.course.hometask3;
+
+/**
+ * Hierarchy of classes: Book, Author, Publisher,
+ * with the method to go through the array and
+ * display books by one of the criteria
+ *
+ * @author Tatyana Nadonenko
+ * @version 1.0
+ * @since 05-04-2023
+ */
+public class Main {
+    public static void main(String[] args) {
+        Book[] arr = new Book[5];
```

Comment:
Here we are going to create Books with Authors and Publishers passed as parameters.